### PR TITLE
Search Summer Cleanup

### PR DESCRIPTION
**Pull request by leogrim**
@eishay @andrewconner @yingjieMiao 

nuke

### CODE REVIEW

File: kifi-backend/modules/cortex/test/com/keepit/cortex/article/FakeCortexArticleProvider.scala [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import com.keepit.common.db.Id
+import com.keepit.model.NormalizedURI
+import com.keepit.search.Lang
+
+class FakeCortexArticleProvider extends CortexArticleProvider {
+  private var articles = Map[Id[NormalizedURI], CortexArticle]()
+  def setArticle(uriId: Id[NormalizedURI], article: CortexArticle): Unit = articles += (uriId -> article)
+  def setArticle(uriId: Id[NormalizedURI], content: String, lang: Lang = Lang("en")): Unit = setArticle(uriId, BasicCortexArticle(Some(lang), content))
+  def getArticle(uriId: Id[NormalizedURI]): Option[CortexArticle] = articles.get(uriId)
+}
```
Comment (leogrim) [inline kifi-backend/modules/cortex/test/com/keepit/cortex/article/FakeCortexArticleProvider.scala:12]: @yingjieMiao

File: kifi-backend/modules/cortex/app/com/keepit/cortex/models/word2vec/Word2VecRepresenters.scala [hunk 2]
```diff
@@ -1,27 +1,44 @@
...
+      articlesById.get(uri.id.get).collect {
+        case articles if articles.nonEmpty =>
+          new ArticleContentExtractor(articles)
+      }
+    }
+  }
+
+  private def getArticle(uri: NormalizedURI): Option[ArticleContentExtractor] = {
+    monitoredAwait.result(getFutureArticle(uri), 10 seconds, s"Get content from Rover for uri: $uri")
+  }
```
Comment (leogrim) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/models/word2vec/Word2VecRepresenters.scala:41]: @yingjieMiao

Comment (yingjieMiao) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/models/word2vec/Word2VecRepresenters.scala:41]: :+1:

**Comment by andrewconner** [comment]
Whoa.

**Pull request merged**